ci-automation/ci_automation_common.sh

```diff
@@ -475,3 +475,27 @@ function list_files() {
     done
 }
 # --
+
+# Looks for ../scripts.patch, ../overlay.patch, ../portage.patch and
+# applies them to the current repo or the respective sub-module checkout.
+function apply_local_patches() {
```

Comment:
Should this function have a check if these patches were already applied?

Reply:
Will add

Reply:
Did a rerun with the changes in http://jenkins.infra.kinvolk.io:8080/job/container/job/packages_all_arches/640/console